intelmq/bots/experts/aggregate/expert.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+import time
+import json
+from intelmq.lib.bot import Bot
+from intelmq.lib.utils import parse_relative
+from intelmq.lib.mixins import CacheMixin
+
+
+class AggregateExpertBot(Bot, CacheMixin):
+    """Aggregation expert bot"""
+
+    fields: str = "classification.type, classification.identifier"
+    threshold: int = 10
+    redis_cache_db: int = 8
+    redis_cache_host: str = "127.0.0.1"  # TODO: could be ipaddress
+    redis_cache_password: str = None
+    redis_cache_port: int = 6379
+    redis_cache_ttl: int = 86400
```

Comment:
Either
- It it should be noted in the docs, that the TTL must be higher than the timespan, otherwise data is lost
- or the TTL is not used at all

Reply:
TTL is currently not used at all. Removing it